service/src/test/java/uk/gov/mca/beacons/api/RetryUtils.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.mca.beacons.api;
+
+import java.util.function.Supplier;
+
+public class RetryUtils {
+
+  public static <T> T runWithRetries(int retries, Supplier<T> run)
+    throws Throwable {
+    try {
+      return run.get();
+    } catch (Throwable e) {
+      if (retries > 0) {
+        Thread.sleep(5000);
+        return runWithRetries(retries - 1, run);
```

Comment:
Can we log a nice visible warning above this, so that we are more likely to see when it retries.